safechef/safechef-v1.context.jsonld

```diff
@@ -0,0 +1,30 @@
+{
+  "@context": {
+    "@version": 1.1,
+    "@protected": true,
+
+    "safechef": "https://credential-handler.github.io/vc-examples/safechef/#",
```

Comment:
```suggestion
    "safechef": "https://credential-handler.github.io/vc-examples/safechef/#",
```

We try not to use CURIEs in context files these days, just full URLs everywhere.

Reply:
Because?